### PR TITLE
feature: Make config installation dir configurable

### PR DESCRIPTION
**Pull request by tvenieris**

### CODE REVIEW

File: roles/blackbox_exporter/defaults/main.yml [hunk 1]
```diff
@@ -65,3 +65,6 @@ blackbox_exporter_configuration_modules:
 #      preferred_ip_protocol: ip6
 #      validate_answer_rrs:
 #        fail_if_matches_regexp: [test]
+
+# Where to put the blackbox_exporter.yml main configuration file
+blackbox_exporter_conf_dir: /etc
```
Comment (SuperQ) [inline roles/blackbox_exporter/defaults/main.yml:70]: Let's be consistent with `conf` vs `config` on all roles.

```suggestion
blackbox_exporter_config_dir: /etc
```

**Review by SuperQ — CHANGES_REQUESTED** [review]

**Comment by SuperQ** [comment]
This also needs to update the `meta/argument_specs.yml` in order to update the auto-generated role docs.

File: roles/blackbox_exporter/defaults/main.yml (hunk 1 above)
Comment (tvenieris) [reply roles/blackbox_exporter/defaults/main.yml]: Changes applied. Apologies for not knowing the purpose of the meta/argument_specs.yml file.

File: roles/blackbox_exporter/defaults/main.yml [hunk 2]
```diff
@@ -68,3 +68,6 @@ blackbox_exporter_configuration_modules:
 #      preferred_ip_protocol: ip6
 #      validate_answer_rrs:
 #        fail_if_matches_regexp: [test]
+
+# Where to put the blackbox_exporter.yml main configuration file
+blackbox_exporter_config_dir: /etc
```
Comment (SuperQ) [inline roles/blackbox_exporter/defaults/main.yml:73]: Bleh, we should probably fix this to match the other roles.

**Review by SuperQ — APPROVED** [review]

File: roles/blackbox_exporter/defaults/main.yml (hunk 2 above)
Comment (SuperQ) [reply roles/blackbox_exporter/defaults/main.yml:73]: Followup issue: https://github.com/prometheus-community/ansible/issues/178

**Pull request merged**